atomicapp/nulecule/base.py

```diff
@@ -548,4 +550,7 @@ def _get_artifact_paths_for_path(self, path):
                 if dir_child.startswith('.') or os.path.isdir(dir_child_path):
                     continue
                 artifact_paths.append(dir_child_path)
+        else:
+            raise NuleculeException("Unable to find artifact %s" % path)
+
```

Comment:
so right now we only raise an exception if the file or dir doesn't exist. what happens if they pass us a directory but the directory is empty? Should that be an error?